### PR TITLE
Complete javadocs documentation for some classes

### PR DESCRIPTION
**Pull request by nikitakovaliov92**

### CODE REVIEW

File: core/src/main/java/org/verapdf/pdfa/validation/profiles/Profiles.java [hunk 1]
```diff
@@ -311,25 +311,20 @@ public static Rule ruleFromValues(final RuleId id, final String object, final St
...
-	 *            a JavaScript expression that is the test carried out on a
-	 *            model instance
-	 * @param error
-	 *            the {@link ErrorDetails} associated with the{@link Rule}.
-	 * @param references
-	 *            a list of further {@link Reference}s for this rule
+	 * @param id          the {@link RuleId} id for the {@link Rule}
+	 * @param object      a String that identifies the Object that the rule applies to
+	 * @param deferred    a Boolean that identifies the deferred property of the rule
+	 * @param tags		  a String that identifies the tag that the rule applies to
```
Comment (MaximPlusov) [inline core/src/main/java/org/verapdf/pdfa/validation/profiles/Profiles.java:317]: a String that contains comma-separated list of tags associated with this rule

File: core/src/main/java/org/verapdf/processor/plugins/Attribute.java [hunk 2]
```diff
@@ -2,16 +2,16 @@
  * This file is part of veraPDF Library core, a module of the veraPDF project.
  * Copyright (c) 2015-2024, veraPDF Consortium <info@verapdf.org>
  * All rights reserved.
- *
+ * <p>
```
Comment (MaximPlusov) [inline core/src/main/java/org/verapdf/processor/plugins/Attribute.java:5]: Please remove these <p>

File: xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java [hunk 3]
```diff
@@ -66,6 +93,11 @@ public String getNamespaceURI() {
         return null;
     }
 
+    /**
+     * Get prefix of xmp node.
```
Comment (MaximPlusov) [inline xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java:97]: Gets

File: xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java [hunk 4]
```diff
@@ -51,12 +65,25 @@ public VeraPDFXMPNode getPropertiesNode() {
...
+     */
+    public void addExtensionSchemaProperty(VeraPDFExtensionSchemaProperty propertyDefinitionXMPNode)
+            throws XMPException {
         if (this.xmpNode != null) {
             getPropertiesNode().getOriginalNode().addChild(propertyDefinitionXMPNode.getXmpNode());
         }
     }
 
+    /**
+     * Get namespaces URI of xmp node.
```
Comment (MaximPlusov) [inline xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java:83]: Gets

**Review by MaximPlusov — CHANGES_REQUESTED** [review]

File: core/src/main/java/org/verapdf/pdfa/validation/profiles/Profiles.java (hunk 1 above)
Comment (nikitakovaliov92) [reply core/src/main/java/org/verapdf/pdfa/validation/profiles/Profiles.java:317]: fixed

File: core/src/main/java/org/verapdf/processor/plugins/Attribute.java (hunk 2 above)
Comment (nikitakovaliov92) [reply core/src/main/java/org/verapdf/processor/plugins/Attribute.java:5]: this <p> is required here

File: xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java (hunk 3 above)
Comment (nikitakovaliov92) [reply xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java:97]: fixed

File: xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java (hunk 4 above)
Comment (nikitakovaliov92) [reply xmp-core/src/main/java/org/verapdf/xmp/impl/VeraPDFExtensionSchemaDefinition.java:83]: fixed

File: core/src/main/java/org/verapdf/processor/plugins/Attribute.java (hunk 2 above)
Comment (nikitakovaliov92) [reply core/src/main/java/org/verapdf/processor/plugins/Attribute.java:5]: pic

**Review by MaximPlusov — APPROVED** [review]

**Pull request merged**